packages/cli/src/commands/init/init.ts

```diff
@@ -56,6 +58,30 @@ interface TemplateOptions {
   installCocoaPods?: string | boolean;
 }
 
+const YARN_VERSION = '3.6.4';
```

Comment:
Do we need to pin this to a specific version?

Reply:
Hm, for me to use `3.x` I need to run first `yarn set version berry` and then I can do `yarn set version 3.x`, I don't know why it happens.

Reply:
Maybe lets add a comment here as context for future readers